assembly/std/address.ts

```diff
@@ -70,8 +70,8 @@ export class Address implements Valider {
    *
    * @return {Address}
    */
-  static fromByteString(bs: string): Address {
-    return new Address(bs);
+  static fromByteString(bs: string, isValid: bool = true): Address {
```

Comment:
why does it return always true ?

Reply:
not always true, the default value of the argument is true, is it was before but I need a way to change it to false during de-serialization